package/doc/sphinx/source/documentation_pages/analysis/parallelization.rst

```diff
@@ -0,0 +1,360 @@
+.. -*- coding: utf-8 -*-
+
+.. _parallel-analysis:
+   
+=================
+Parallel analysis
+=================
+
+.. versionadded:: 2.8.0
+   Parallelization of analysis classes was added during Google Summer of Code
+   2023 by @marinegor and MDAnalysis GSoC mentors.
```

Comment:
```suggestion
   2023 by @marinegor and MDAnalysis GSoC mentors.
```
Maybe add an actual link to your GitHub profile? (I don't think this is linked automatically, but might be wrong.)

Reply:
Added, thanks. I think you're right, it's not linked automatically (at least online or in vscode)